src/inference/src/threading/ie_cpu_streams_executor.cpp

```diff
@@ -98,6 +98,7 @@ struct CPUStreamsExecutor::Impl {
                     // wrapping around total_streams (i.e. how many streams all different core types can handle
                     // together)
                     const auto total_streams = _impl->total_streams_on_core_types.back().second;
+                    const auto core_type_size = _impl->total_streams_on_core_types.size();
```

Comment:
There are lots of "core_type_size>0", why not use a flag to instead it? such as:
`const auto hybrid_core = _impl->total_streams_on_core_types.size() > 1;`

Reply:
Ok, I fixed it according to your proposal.